docs/network-map/filtering-search.md

```diff
@@ -0,0 +1,104 @@
+title: Filtering & Search
```

Comment:
Filtering & Search ==> Network Map Filtering & Search
(this title is what shows up in e.g. SERPs)